tools/build.js

```diff
@@ -13,6 +13,7 @@ mkdirp('dist')
     .then(() => commandRunner('npm run build-editor'))
     .then(() => mkdirp('dist/editor'))
     .then(() => copy('./src/editor/dist/**', './dist/editor'))
+    .then(() => commandRunner('parcel build src/editor/src/routes.js --out-dir dist/editor'))
```

Comment:
You don't need to explicitly build `routes.js` since they are included in the `service-worker.js`